interactive_engine/executor/ir/proto/algebra.proto

```diff
@@ -41,6 +47,8 @@ message Project {
   // An indicator to tell the runtime whether the projected value is appending to or replacing
   // existing relation.
   bool is_append = 2;
+  // The datatype of output results
+  repeated MetaData op_meta = 3;
```

Comment:
Why we donot use map<alias, IrDataType> for MetaData? As before, think meta_data is a better name than op_meta

Reply:
Have renamed as meta_data. `repeated MetaData` here is defined in consist with, e.g., `repeated ExprAlias` in `Project`.